public/app/features/alerting/unified/Receivers.tsx

```diff
@@ -12,7 +11,7 @@ const EditContactPoint = SafeDynamicImport(() => import('./components/contact-po
 const NewContactPoint = SafeDynamicImport(() => import('./components/contact-points/NewContactPoint'));
 const GlobalConfig = SafeDynamicImport(() => import('./components/contact-points/components/GlobalConfig'));
 
-const ContactPoints = (_props: GrafanaRouteComponentProps): JSX.Element => (
```

Comment:
I know this is not used now, but are we sure we want to delete these _props in this pr? maybe it was here on purpose for some reason?

Reply:
Fair question - I removed it here as in the tests I then have to pass in all sorts of route props (route, queryParams, history, location, match), and they weren't used before this anyway. I had to make a similar change in this PR https://github.com/grafana/grafana/pull/88487/files#diff-5e4993ba6405cb0c042ccd8d131bd81d954ff9a86d4b854b1cc2e7f095f798ab